### PR TITLE
i#2-Code Refactoring: Project Initialization

### PR DESCRIPTION
**Pull request by yliang123**
- [x] Add eslint google style check
- [x] Initialize tailwand css
- [x]  Initialize by react-create-app
Issues: #2

### CODE REVIEW

**Review by dolanzhao — COMMENTED** [review]
Test

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
```
Comment (dolanzhao) [inline .gitignore:5]: Test

Comment (yliang123) [reply .gitignore:5]: 1111

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (dolanzhao) [inline README.md:70]: Clear this file.

File: postcss.config.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  plugins: {
+    tailwindcss: {},
```
Comment (dolanzhao) [inline postcss.config.js:3]: Add comments for this file.

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
```
Comment (dolanzhao) [inline public/index.html:41]: Delete auto-generated comments.

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
```
Comment (dolanzhao) [inline public/index.html:16]: Same as https://github.com/xperflab/web/pull/1/files#r904450187

File: public/index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
```
Comment (dolanzhao) [inline public/index.html:26]: Same as https://github.com/xperflab/web/pull/1/files#r904450187

File: src/App.js [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+    } else if (this.state.showComponent === "VR") {
+      renderComponent = (<VR />)
+    } else if (this.state.showComponent === "treetable") {
+      renderComponent = (<Treetable />)
+    }
+    const { Header, Sider, Content } = Layout;
+    return (
+
+
+      <>
```
Comment (dolanzhao) [inline src/App.js:178]: Fix this.

File: .eslintrc.json [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+        },
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "import"
+    ],
+    "rules": {
+        "linebreak-style": [0, "windows"],
```
Comment (dolanzhao) [inline .eslintrc.json:22]: Use default setting.

File: public/index.html [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/easyview_logo.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (dolanzhao) [inline public/index.html:12]: Update this line.

File: public/manifest.json [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
```
Comment (dolanzhao) [inline public/manifest.json:19]: Remove unuse res.

File: src/App.js [hunk 11]
```diff
@@ -0,0 +1,198 @@
...
+          }
+        </div>
+      </div>
+
+
+    );
+  }
+}
+
+export default App;
```
Comment (dolanzhao) [inline src/App.js:198]: Keep only the init code.

**Review by dolanzhao — APPROVED** [review]

**Pull request merged**